src/classes/MODL_PrimaryAfflMapper.cls

```diff
@@ -52,7 +53,13 @@ public class MODL_PrimaryAfflMapper {
     * @param settingsRecordTypeValue 'Account Record Type' value from EDA Settings.
     * @param isValid Boolean flag to indicate whether the Record Type Affiliation maps to a valid Account record type.
     *******************************************************************************************************/ 
-    public MODL_PrimaryAfflMapper(Schema.RecordTypeInfo accountRecordTypeInfo, Schema.DescribeFieldResult contactDescribeFieldResult, String settingsContactFieldValue, String settingsRecordTypeValue, Boolean isValid) {
+    public MODL_PrimaryAfflMapper(Schema.RecordTypeInfo accountRecordTypeInfo, 
```

Comment:
Expanding the constructors here was the correct call. We now have another required field for a valid mapping that allows us to inform the user of the specific validation failure.